### ISSUE TITLE
embetter paper to dos

### ISSUE THREAD

**Issue by wrgr** [openconnectome/ocp-journal-paper#59]
### use case 2: spatial synapse reproducibility
- [x] initial rewrite of paper section for content and possibly ocar
- [x] get analysis code from jordan 
- [x] get latest mask from dean with description of approach (using all layers for now)
- [x] upload mask to neurodata
- [x] build reproducible pipeline (script) to generate synapse count matrix from source data | vesicle run and mask upload
- [x] apply initial analysis code (extracting new count matrix)
- [x] rewrite content to reflect the above
### tables and website
- [x] table 2: guess approximate object count and put answers in (a <5 minute exercise)
- [x] reproducability: move [reproducability](https://github.com/neurodata/ndpaper/blob/master/store/store.md) to "correct" location @willgray to tell kunal what it is.
### case study 1
- [x] rewrite of code for content and light ocar revision
- _other inputs to go here as time permits.  Case study 2 this week, CS 1 cleanup next week._

**Comment by jovo** [comment]
i gave jordan (and the class) the data that i had that had been binned.
i can check my old emails and see if i know which one.
in theory, we should be able to count bins and figure it out?

On Friday, June 10, 2016, William Gray notifications@github.com wrote:

> Assigned #59 https://github.com/neurodata/ndpaper/issues/59 to @jovo
> https://github.com/jovo.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/neurodata/ndpaper/issues/59#event-688482515, or mute
> the thread
> https://github.com/notifications/unsubscribe/AACjcvlkm2uJbQFW4Q9uKG4cus3fGzVIks5qKVNOgaJpZM4Iy46P
> .

## 

the glass is all full:  half water, half air.
neurodata.io, jovo calendar
https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York

**Comment by jovo** [comment]
@willgray updated this task with your other one, so issues are all person specific.

**Comment by jovo** [comment]
@willgray please add DoD for your tasks, i'm confused what will be with regard to the use cases by monday.  afaict, all the rest of the contents are in place for the whole manuscript.

**Comment by wrgr** [comment]
you were moving issues, so i was waiting on you so as to not interfere.  :)  seems like things are ready now.  will be more explicit on dod.

**Comment by jovo** [comment]
perfect, thank you.

On Wed, Jun 15, 2016 at 6:50 AM, William Gray notifications@github.com
wrote:

> you were moving issues, so i was waiting on you so as to not interfere. :)
> seems like things are ready now. will be more explicit on dod.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/neurodata/ndpaper/issues/59#issuecomment-226153262,
> or mute the thread
> https://github.com/notifications/unsubscribe/AACjct991GQ0pAppP-Cbn-II0jY9qSU6ks5qL9jngaJpZM4Iy46P
> .

## 

the glass is all full:  half water, half air.
neurodata.io, jovo calendar
https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York

**Comment by wrgr** [comment]
@jovo - updated.  upshot is my inputs for others, cs2 cleanup, which was the gnarliest remaining thing this week, and cs1 next week.

**Comment by wrgr** [comment]
This is much improved, but more to do.  Declaring success and starting a new issue!  To tell you more tomorrow.
